doc/version_history.rst

```diff
@@ -6,6 +6,14 @@
 Version History
 ##################
 
+.. _lsst.ts.m2gui-0.5.2:
+
+-------------
+0.5.2
```

Comment:
I think this needs to be 0.6.0 as you are removing some methods from the `Model` class.

Reply:
Fixed in the commit: [f6c6861](https://github.com/lsst-ts/ts_m2gui/pull/46/commits/f6c68611b5e1da142fd9522b16702665144cdfcb). Thanks!